jiant/tasks/seq2seq.py

```diff
@@ -21,15 +22,19 @@
 )
 
 
-@register_task("seg_wix", rel_path="seg/wix/", max_targ_v_size=200)
+@register_task("seg_wix", rel_path="seg/wix/", max_targ_v_size=200, valid_metric="accuracy")
+@register_task("wmt14_en_de", rel_path="wmt14/en_de/", max_targ_v_size=40000, valid_metric="BLEU")
 class Seq2SeqTask(SequenceGenerationTask):
     """Sequence-to-sequence Task"""
 
-    def __init__(self, path, max_seq_len, max_targ_v_size, name, **kw):
+    def __init__(self, path, max_seq_len, max_targ_v_size, name, valid_metric, **kw):
         super().__init__(name, **kw)
-        self.scorer2 = BooleanAccuracy()
+        if valid_metric == "accuracy":
```

Comment:
Make it clear in the header comment for this task class what the options are for this argument. (And generally spell out what all the arguments are.)

Reply:
(And what this task class is and isn't meant to support.)

Reply:
I added a comment that this comes from the config file. The options are listed there.